### PR TITLE
D888

### PR DESCRIPTION
**Pull request by val-fed**
PR description

### CODE REVIEW

File: code_review.proto [hunk 1]
```diff
@@ -57,47 +57,47 @@ message RemoveWorkspaceRequest { string workspace_name = 1; }
...
-  // others.
-  Author author = 3;
-  repeated Reviewer reviewer = 4;
-  string description = 5;
-  string bug = 6;
-  Status status = 7;
-
-  string workspace = 8;
-  int64 created_timestamp = 9;
+	message Diff {
```
Comment (val-fed) [inline code_review.proto]: 60 60 60 60 60 60 60 60 60 60 60 60 60 60 60 60

File: code_review.proto [hunk 2]
```diff
@@ -57,47 +57,47 @@ message RemoveWorkspaceRequest { string workspace_name = 1; }
...
-  string email = 1;
-  bool needs_attention = 2;
-}
+	  // Threads on a line.
+	  repeated Thread code_thread = 13;
+	  repeated string cc = 14;
+	  // Threads on the whole diff, not on a particular file or line.
+	  repeated Thread diff_thread = 15;
+	}
+
```
Comment (val-fed) [inline code_review.proto]: 90 90 90 90 90 90 90 90 90 90 90 90 90 90 90 90 90 90 90 90 90 90 90

File: code_review.proto [hunk 3]
```diff
@@ -125,4 +125,4 @@ message Comment {
   string content = 1;
   int64 timestamp = 2;
   string created_by = 3;
-}
+	}
```
Comment (val-fed) [inline code_review.proto]: 128

File: code_review.proto (hunk 1 above)
Comment (valerii-fed) [inline code_review.proto]: Cocogamgo 60

File: code_review.proto (hunk 3 above)
Comment (valerii-fed) [inline code_review.proto]: 128 Coco

File: code_review.proto [hunk 4]
```diff
@@ -57,56 +57,32 @@ message RemoveWorkspaceRequest { string workspace_name = 1; }
...
 
-// Comment thread. Note that for whole-diff threads, some fields will be empty
-// (e.g repo_id, commit_id).
-message Thread {
-  enum Type {
-    // Thread on a code line
-    CODE = 0;
-    // Thread on commit (synced from other systems e.g GitHub)
-    COMMIT = 1;
     // Thread on repo (synced from other systems e.g GitHub)
```
Comment (val-fed) [inline code_review.proto]: line 86 RIGHT

File: new_file.txt [hunk 5]
```diff
@@ -0,0 +1,11 @@
+11111
+2222222222222
+333333333333
+444444444444444
```
Comment (val-fed) [inline new_file.txt]: 444444444444444444444444444444444444444444444444444444444444444444444444

File: README.md [hunk 6]
```diff
@@ -1,23 +1,23 @@
...
+	# test-repo
+	test1
+	test2
+	test3
+	test4
+	test5
+	test6
+	test7
+	test8
+	test9
```
Comment (valerii-fed) [inline README.md]: new 10 10 10 10 10 10 10 10 10 10 10 10 10 10 10 10 10 10 10 10 10 10 10 10 10 10 10 10 10 10 10 10 10

**Comment by valerii-fed** [comment]
This is PR comment!

File: code_review.proto (hunk 1 above)
Comment (valerii-fed) [reply code_review.proto]: Answer to 60 60 60 60 60 60 60 60 60 60 60 60 60 60 60 60 60 60

File: code_review.proto [hunk 7]
```diff
@@ -57,47 +57,47 @@ message RemoveWorkspaceRequest { string workspace_name = 1; }
...
-  Status status = 7;
-
-  string workspace = 8;
-  int64 created_timestamp = 9;
+	message Diff {
+	  enum Status {
+	    REVIEW_NOT_STARTED = 0;
+	    NEEDS_MORE_WORK = 1;
+	    UNDER_REVIEW = 2;
+	    ACCEPTED = 3;
```
Comment (valerii-fed) [inline code_review.proto]: line 65 65  65 65 65 65 65 65 65 65 65 65 65 65

File: new_file.txt [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+55555555555555555
+666666666666
+7777777777777777
+8888888888888888
+9999999999999999
```
Comment (valerii-fed) [inline new_file.txt]: 9999999999999999999999999999999999999999999999999999

**Comment by val-fed** [comment]
PR comment!!!

**Pull request closed without merging**